test/adminTestRoutes.js

```diff
@@ -719,6 +720,23 @@ describe("Add a menu by Admin", () => {
 
 describe("Polulate database", () => {
 
+	// it("should return 404 if all users doesn't exist", (done) => {
```

Comment:
#### Nitpick
If you don't need a particular code block or line of code, delete it. There's no point commenting it out.